gateway/src/main/kotlin/Command.kt

```diff
@@ -80,7 +80,7 @@ public sealed class Command {
 
 @Serializable
 internal data class Identify(
-    internal val token: String,
```

Comment:
reason: this doesn't provide any safety, internal visibility becomes `public` on jvm anyway